### PR TITLE
fix: customer profile update

### PR DESCRIPTION
**Pull request by rugvedcbt**
fixed user profile update error in customer pages. Added no data animation in club search screen.

### CODE REVIEW

**Review by ramprasadm-CBT — APPROVED** [review]

File: backend/frontend/src/screens/LoginScreen.js [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
-    } else if(!userInfo && error && password.length > 0){
-      setErrorMessage("Incorrect Password")
-      setPassword("")
+      setErrorMessage("");
+    } else if (!userInfo && LoginError && password.length > 0) {
+      setErrorMessage("Incorrect Password");
+      setPassword("");
     }
-  }, [navigate, userInfo, dispatch, error]);
+  }, [navigate, userInfo, dispatch, LoginError]);
```
Comment (murucbt) [inline backend/frontend/src/screens/LoginScreen.js]: Nit: ASC order.

File: backend/frontend/src/screens/LoginScreen.js [hunk 2]
```diff
@@ -98,8 +98,10 @@ function LoginScreen() {
 
               <label>Password</label>
               <TextField
-                error = {error && errorMessage.length > 0}
-                helperText={ errorMessage.length > 0 && error && errorMessage }
+                error={LoginError && errorMessage.length > 0}
+                helperText={
+                  errorMessage.length > 0 && LoginError && errorMessage
+                }
```
Comment (murucbt) [inline backend/frontend/src/screens/LoginScreen.js:104]: Is this helpText is a boolean value or a string value? will this give string result or boolean result? `errorMessage.length > 0 && LoginError && errorMessage`

File: backend/frontend/src/screens/ClubSearchScreen.js [hunk 3]
```diff
@@ -98,14 +105,16 @@ function ClubSearchScreen() {
...
-      </div>
-      <div className="clubs-error">
-        {filteredClubLocations?.length === 0 && <h2>No clubs available</h2>}
-      </div>
+        </div>)
+        :
+        (<div className="clubs-error">
+          <NoDataAnimation url={NoDataAnimationUrl} />
+        </div>)
+        }
```
Comment (murucbt) [inline backend/frontend/src/screens/ClubSearchScreen.js]: This html section has some alignment issue. check once.

File: backend/frontend/src/screens/LoginScreen.js (hunk 2 above)
Comment (rugvedcbt) [reply backend/frontend/src/screens/LoginScreen.js:104]: LoginError is boolean value from redux state, if it is true then a error string value is set to errorMessage then it is given to helpertext. helpertext does not display if the errorMessage is empty.

**Pull request merged**